### PR TITLE
Add ecosystem page

### PR DESCRIPTION
**Pull request by dgca**
### What changed?

- Added ecosystem page

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

### CODE REVIEW

**Comment by skylarrichard123** [comment]
Looking awesome @dgca 

(1) The navigation copy can be: "Our vast ecosystem offers diverse apps and integrations to choose from."
(2) Let's do title case for "How will you use Iron Fish?" -> "How Will You Use Iron Fish?"
(3) There's about 8px padding between those filters when they're on the same row. Could we nudge that up to 12px?

I'll press Elena for that list.

File: content/ecosystem/ecosystem.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  },
+  {
+    type: "Mining Pools",
+    name: "GPU Mine",
+    link: "https://gpumine.org/",
+    image: "/images/ecosystem/gpu-mine.png",
+  },
+  {
+    type: "Mining Pools",
+    name: "UnMineable",
```
Comment (dguenther) [inline content/ecosystem/ecosystem.ts:108]: I think they capitalize it like `unMineable`: https://unmineable.com/coins

File: content/ecosystem/ecosystem.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  },
+  {
+    type: "Mining Pools",
+    name: "Ezil",
+    link: "https://ezil.me/",
+    image: "/images/ecosystem/ezil.png",
+  },
+  {
+    type: "Mining Pools",
+    name: "GPU Mine",
```
Comment (dguenther) [inline content/ecosystem/ecosystem.ts:102]: I think this is GPUMINE, like one word: https://gpumine.org/en

File: content/ecosystem/ecosystem.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  },
+  {
+    type: "Mining Pools",
+    name: "Herominers",
+    link: "https://herominers.com/",
+    image: "/images/ecosystem/herominers.png",
+  },
+  {
+    type: "Mining Pools",
+    name: "ZkWork",
```
Comment (dguenther) [inline content/ecosystem/ecosystem.ts:54]: This is ZK.Work: https://twitter.com/ZKWorkHQ

**Review by dguenther — APPROVED** [review]
The frontend code looks good, just added a few rename suggestions!

**Pull request merged**